### DISCUSSION TITLE
话说这个留名字的挑战

### DISCUSSION THREAD

**Discussion by gudaoxifengshouma** [ByteLegendQuest/remember-brave-people#209] [Q&A]
图片已经看不了了，自己摸索完全没有头绪

**Comment by blindpirate** [comment]
图片不是看不了，而是raw.githubusercontent.com域名被中国大陆屏蔽了。稍后我们会上线代理功能，把文档里的图片通过我们的服务器中转。

**Reply by gudaoxifengshouma** [reply]
感谢大佬在百忙之中的回复，由于图片被屏蔽了，如图所示，我目前就卡在这一步，不知为何无法点击propose，我是一名对编程感兴趣的小白，无意之中在知乎上看到了您的游戏，希望能通过工作之余的学习慢慢的将您这款前途无限的游戏通关，再次感谢您对我的回复。
![QQ图片20210910100324](https://user-images.githubusercontent.com/90312351/132787179-8d4b3d4e-0da9-4f84-9bd4-d738da47996a.png)

**Reply by blindpirate** [reply]
代理功能已经上线，从游戏中访问教程的图片会通过服务器中转。

**Reply by cai895015380** [reply]
不好意思，请问游戏教程里可以提示如何解决
You’re making changes in a project you don’t have write access to. We’ve created a fork of this project for you to commit your proposed changes to. Submitting a change will write it to a new branch in your fork, so you can send a pull request.
吗？
我这样的纯小白卡在了这一步，只能麻烦大佬们了！

**Reply by blindpirate** [reply]
> You’re making changes in a project you don’t have write access to. We’ve created a fork of this project for you to commit your proposed changes to. Submitting a change will write it to a new branch in your fork, so you can send a pull request.

这不是问题，无需解决——它不是一个错误，只是一个提示信息，如果你仔细看教程的话，你会发现教程里有一模一样的提示。

![image](https://user-images.githubusercontent.com/12689835/132942549-fb8df3f5-e3be-44d6-a0be-2db6302f0453.png)

**Reply by cai895015380** [reply]
是的，我确实拖到了最后面，但是我的提交按钮propose changes是灰色的，它并不能点击，很抱歉我的叙述不周导致的问题，希望您能够理解。


它提示我并没有权限进行操作，所以我不得不再次打扰您

**Reply by cai895015380** [reply]
> > You’re making changes in a project you don’t have write access to. We’ve created a fork of this project for you to commit your proposed changes to. Submitting a change will write it to a new branch in your fork, so you can send a pull request.
> 
> 这不是问题，无需解决，你真的仔细看教程了么？教程说拖动滚动条到最下面，你拖了么？
> 
> ![image](https://user-images.githubusercontent.com/12689835/132942549-fb8df3f5-e3be-44d6-a0be-2db6302f0453.png)

很不好意思，由于没有合适的VPN我的图片发不出来，但是情况和上面那个人是一样的

**Reply by cai895015380** [reply]
> ![QQ图片20210910100324](https://user-images.githubusercontent.com/90312351/132787179-8d4b3d4e-0da9-4f84-9bd4-d738da47996a.png)

和这个图片里面的情况一样

**Reply by blindpirate** [reply]
请 仔 细 阅 读 教 程

教程说，

> 从现有的文件中拷贝一段，注意以逗号开头，花括号结尾！！非常重要！！
> 粘贴，并修改其中的坐标，颜色和GitHub ID。注意，非常重要！！
> 。。。。

是否已经完成？

**Reply by cai895015380** [reply]
我做了，甚至非常认真的检查是否存在坐标的重复和GitHub ID是否有问题，但是提交按钮仍然显示为不能点击，如果填写格式问题会导致提交按钮不能点击的话，我希望您可以回复我一个‘是’，毕竟如果填写格式问题会导致提交按钮不能点击的条件是成立的，那么就很有可能是我自己的代码格式问题。
不好意思再次打扰您了！还请您容忍新手的各种生疏之处！

**Reply by blindpirate** [reply]
请将你的截图，全屏，发到`bo@bytelegend.com`。

**Reply by cai895015380** [reply]
> 请将你的截图，全屏，发到`bo@bytelegend.com`。

是通过邮箱吧?

**Reply by blindpirate** [reply]
是的
